lighthouse-core/report/v2/report-generator.js

```diff
@@ -18,19 +18,24 @@
 
 const fs = require('fs');
 
-const REPORT_TEMPLATE = fs.readFileSync(__dirname + '/report-template.html', 'utf8');
+const readFile = filename => fs.readFileSync(__dirname + filename, 'utf8');
+const sanitizeScript = src => src.replace(/<\//g, '\\u003c/');
+const readScript = filename => '<script>' + sanitizeScript(readFile(filename)) +
+    `\n//# sourceURL=file://${__dirname}${filename} \n</script>`;
+
+const REPORT_TEMPLATE = readFile('/report-template.html');
 const REPORT_JAVASCRIPT = [
-  fs.readFileSync(__dirname + '/renderer/util.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/dom.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/details-renderer.js', 'utf8'),
-  fs.readFileSync(__dirname + '/../../lib/file-namer.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/logger.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/report-ui-features.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/category-renderer.js', 'utf8'),
-  fs.readFileSync(__dirname + '/renderer/report-renderer.js', 'utf8'),
-].join(';\n');
-const REPORT_CSS = fs.readFileSync(__dirname + '/report-styles.css', 'utf8');
-const REPORT_TEMPLATES = fs.readFileSync(__dirname + '/templates.html', 'utf8');
+  readScript('/renderer/util.js'),
```

Comment:
unrelated, this is kosher with browserify? I think we have some other comments and verbose lines to remove :)